portal/templates/cluster_node_details.html

```diff
@@ -0,0 +1,63 @@
+{%extends "base.html"%}
+
+{%block title%}
+  Node Details
+{%endblock%}
+
+{%block body%}
+<section id="cluster-node-details" class="bg-light">
+  <div class="container col-lg-12 mx-auto">
+
+    <div class="row">
+      <div class="col-lg-12 mx-auto">
+        <ol class="breadcrumb">
+          <li class="breadcrumb-item"><a href="{{url_for('dashboard')}}">Home</a></li>
+          <li class="breadcrumb-item"><a href="{{url_for('list_clusters')}}">Clusters</a></li>
+          <li class="breadcrumb-item"><a href="/clusters/{{cluster_name}}">{{cluster_name}}</a></li>
```

Comment:
You'll want to change the `href` here to `{{ url_for('view_public_cluster', name=cluster_name) }}` in order to follow flask convention for url redirects. Otherwise, this will simply add `/clusters/{{ cluster_name }}` to the end of the current page's URL, giving you something like `"https://portal.slateci.io/clusters/{{ cluster_name }}/{{ node_name }}/clusters/{{ cluster_name }}"`